contact/templates/contact/submit-success.html

```diff
@@ -0,0 +1,35 @@
+{% extends "core/base.html" %}
+{% load static from staticfiles %}
+{% load message_box from directory_components_tags %}
+
+{% block head_css %}
+    {{ block.super }}
+    <link href="{% static 'styles/contact.css' %}" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block content %}
+
+    {% include 'contact/breadcrumbs.html' %}
+    <div class="container">
+        <section>
+          <div class="container">
+              <div class="grid-row">
+                  <div class="column-two-thirds">
+                      <div id="success-message-container">
+                        {% message_box heading=page.heading description=page.body_text %}
```

Comment:
if this is the main page heading you can use `heading_level='h1'` to the message box component as it defaults to h3